africanlii/templates/africanlii/_document_table.html

```diff
@@ -1,6 +1,6 @@
 {% load i18n %}
 
-<table class="table">
+<table>
```

Comment:
No, we still want the table class so that we apply bootstrap's table styles, but the custom border we have put around `table.table` must be removed.

@musangowope

Reply:
@greg with `<table table-borderless>` it looks like below:
![image](https://user-images.githubusercontent.com/15012985/174800778-d586855a-ca27-4467-9247-8469b15361ed.png)

Reply:
@nickmwangemi 
1. delete `_tables.scss (/africanlii/static/stylesheets/_tables.scss)`
2. delete this [line](https://github.com/laws-africa/peachjam/blob/b66fa96b4767ee2bb6f50777512a6e91bb12190d/africanlii/static/stylesheets/app.scss#L12)

that will remove the styles associated to `table.table`. Also just use `<table class"table">`, we dont need the class `table-borderless`

Reply:
Borderless table

![image](https://user-images.githubusercontent.com/15012985/175009564-1f8f2ddd-9104-4095-8356-7be7f91a2f96.png)